nav2_map_server/test/map_saver_cli/test_map_saver_cli.cpp

```diff
@@ -92,9 +92,9 @@ TEST(MapSaverCLI, CLITest)
     std::string(
     "ros2 run nav2_map_server map_saver_cli "
     "-t map_failure --occ 100 --free 2 --mode trinary --fmt png -f ") + file_path +
-    std::string("--ros-args __node:=map_saver_test_node");
+    std::string(" --ros-args --remap __node:=map_saver_test_node");
   return_code = system(command.c_str());
-  EXPECT_EQ(return_code, 65280);
+  EXPECT_EQ(return_code, 256);
```

Comment:
No note assuming CI test is happy, but noting this for myself to check

Reply:
Ah, it looks like there are bunch of failing external symbols in the tests.  That means that something is wrong with this PR.  I'll take a closer look and see if I can reproduce locally.

Reply:
So I can't reproduce, and looking at the code again there is nothing obviously wrong jumping out at me.

I'm not that familiar with CircleCI, but in the job below it never seems to run the "Setup Workspace" or "Build Workspace" steps that are listed in https://github.com/ros-navigation/navigation2/blob/main/.circleci/config.yml .  Looking at those logs would be helpful for me to try and reproduce.

@SteveMacenski Do you know why those steps aren't present in that run, and/or how I might get access to those logs?

Reply:
The `release_build` job below does the building of the main software stack (`system_build` does the system tests separately). The job that's failing (`release_test`) is only doing the `colcon test` bits on those built and cached artifacts from the builds.

You can find the build artifacts here: https://app.circleci.com/pipelines/github/ros-navigation/navigation2/12291/workflows/ec12a613-f542-4479-9c2b-640744e3fbaf/jobs/37306/artifacts
You can find the test artifacts here: https://app.circleci.com/pipelines/github/ros-navigation/navigation2/12291/workflows/ec12a613-f542-4479-9c2b-640744e3fbaf/jobs/37308/artifacts

I'm also retriggering the build from the start to validate its not a fluke, but it does seem like its a problem

Reply:
Something else worth trying is to update `v26` to `v27` in the 3x places in the circle config file to break the cache which may contain the previous works and rebuild from scratch.

Reply:
Ah, thanks for pointing out where it was built.  Following that, I was able to see that the CircleCI job uses a number of mixins, and compiling locally with those mixins I can reproduce.  Now to see what is going on here.

Reply:
All right, I found the issue.  [66e300b](https://github.com/ros-navigation/navigation2/pull/4557/commits/66e300b43bf5b1a3a5246bd591d3593d211d1a66) should fix it.